asab/storage/elasticsearch.py

```diff
@@ -83,6 +104,48 @@ def session(self):
 		return self._ClientSession
 
 
+	async def is_connected(self) -> bool:
```

Comment:
Why this method is needed?

It is non-systematic - there is no direct connection between the ASAB application and ElasticSearch.